pyccel/ast/operators.py

```diff
@@ -2,8 +2,6 @@
 # This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
 # go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
 #------------------------------------------------------------------------------------------#
-# TODO [EB 12.03.21]: Remove pylint command with PR #797
-# pylint: disable=W0201
```

Comment:
This command also needs removing in bitwise_operators.py

Reply:
I don't see this command in bitwise_operators.py